mmv1/products/kms/EkmConnection.yaml

```diff
@@ -0,0 +1,163 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EkmConnection'
+base_url: 'projects/{{project}}/locations/{{location}}/ekmConnections'
+create_url: 'projects/{{project}}/locations/{{location}}/ekmConnections?ekmConnectionId={{name}}'
+self_link: 'projects/{{project}}/locations/{{location}}/ekmConnections/{{name}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  `Ekm Connections` are used to control the connection settings for an `EXTERNAL_VPC` CryptoKey.
+  It is used to connect customer's external key manager to Google Cloud EKM.
+
+
+  ~> **Note:** Ekm Connections cannot be deleted from Google Cloud Platform.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Creating a Ekm Connection': 'https://cloud.google.com/kms/docs/create-ekm-connection'
+  api: 'https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.ekmConnections'
+id_format: 'projects/{{project}}/locations/{{location}}/ekmConnections/{{name}}'
+import_format: ['projects/{{project}}/locations/{{location}}/ekmConnections/{{name}}']
+skip_delete: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'kms_ekm_connection_basic'
+    primary_resource_id:
+      'example-ekmconnection'
+    skip_test: true
+    vars:
+      ekmconnection_name: 'ekmconnection_example'
+      servicedirectoryservice: 'external-servicedirectoryservice'
+      hostname: 'external-uri'
+      raw_der: 'playground-cert'
+      sec_project_id: '315636579862'
```

Comment:
With `skip_test: true` these values will only be used for user documentation as examples. Do we want to generalize these values to be more "example" like? Specifically the project ID.